lib/web/websocket/stream/websocketstream.js

```diff
@@ -388,7 +388,7 @@ class WebSocketStream {
     // 6. If the connection was closed cleanly ,
     if (wasClean) {
       // 6.1. Close stream ’s readable stream .
-      this.#readableStream.cancel().catch(() => {})
+      this.#readableStreamController.close()
```

Comment:
We should probably handle the rejection of this return value, shouldn't we?
```suggestion
      this.#readableStreamController.close().catch(() => {})
```
Ref: https://streams.spec.whatwg.org/#ws-close

Reply:
That is for a WritableStream, we have a ReadableStreamDefaultController, which returns void and can't throw an error afaict https://github.com/nodejs/node/blob/1effb262b64defdaa8e7d41a08b9cc5e9d0c075d/lib/internal/webstreams/readablestream.js#L1066

Reply:
I apologize, I seem to have misunderstood something.